modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/display/ConfigurationCategoryMenuDisplay.java

```diff
@@ -71,15 +75,8 @@ private void _addConfigurationModel(ConfigurationModel configurationModel) {
 		configurationScopeDisplay.add(configurationModel);
 	}
 
-	private static final String[] _SCOPE_KEYS = {
```

Comment:
The reason for having this constant was to specify the desired order in the UI. Although it probably wasn't obvious. I'm going to think of a better name for the constant  to make it more clear and will add it back. 
I will leave the rest of your changes.